src/main/java/genji/GenjiException.java

```diff
@@ -0,0 +1,11 @@
+package genji;
+
+public class GenjiException extends Exception{
```

Comment:
Add a whitespace after the `Exception` to follow the coding standard.

**For example:**
```java
public class GenjiException extends Exception {
    // Rest of the class
}
```